examples/bri-3/src/bri/state/models/stateContent.ts

```diff
@@ -0,0 +1,28 @@
+import { AutoMap } from '@automapper/classes';
+import { StateLeafValues } from './stateLeafValues';
+import MerkleTree from 'merkletreejs';
+
+export class StateContent {
```

Comment:
what is the difference between StateContent domain object and StateLeafValues domain object? Can we one instead of both?

Reply:
If it is to much work we can also merge with both of these and tackle this comment laters as part of mil5

Reply:
no difference now. One had the state tree the other did not. Switched to a single class called StateTreeLeafValueContent